lib/plugins/aws/package/compile/events/schedule.js

```diff
@@ -166,6 +190,23 @@ class AwsCompileScheduledEvents {
               }
             `;
 
+              const schedulerScheduleTemplate = `
```

Comment:
Let's use upfront object (not string) for CF configuration.

I know it's not consent with how older version is configured, but that one desires the refactor (welcome in other PR)